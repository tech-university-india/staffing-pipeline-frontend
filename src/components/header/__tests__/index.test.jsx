import React from 'react';
import { render } from '@testing-library/react';
import Header from '..';

describe('Header', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Header {...{ hasNav: true }} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
