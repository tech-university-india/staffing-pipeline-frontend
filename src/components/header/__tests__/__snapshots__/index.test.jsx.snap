// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <div
      class="content-padding"
    >
      <img
        class="logo"
        src="./assets/McK_Logo.png"
      />
      <div
        class="navbar"
      >
        <button
          class="nav-button"
        >
          Engagements
        </button>
        <button
          class="nav-button"
        >
          People
        </button>
        <button
          class="nav-button"
        >
          Case Studies
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Header should render correctly with a navbar when Navbar boolean is true 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <div
      class="content-padding"
    >
      <img
        class="logo"
        src="./assets/McK_Logo.png"
      />
      <div
        class="navbar"
      >
        <button
          class="nav-button"
        >
          Engagements
        </button>
        <button
          class="nav-button"
        >
          People
        </button>
        <button
          class="nav-button"
        >
          Case Studies
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Header should render correctly without a navbar when Navbar boolean is false 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <div
      class="content-padding"
    >
      <img
        class="logo"
        src="./assets/McK_Logo.png"
      />
    </div>
  </div>
</DocumentFragment>
`;
